example-adapter/pom.xml

```diff
@@ -213,6 +211,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.owasp</groupId>
+                <artifactId>dependency-check-maven</artifactId>
+                <version>8.1.0</version>
```

Comment:
I got the following warning when running the build:

```
[WARNING] A new version of dependency-check is available. Consider updating to version 8.1.2.
```

It is probably just an update that landed after this PR was created, but it might be worth upgrading since we know one is available.

Reply:
Updated the plugin to version 8.1.2.